IPython/utils/PyColorize.py

```diff
@@ -59,6 +59,21 @@
 else:
     from StringIO import StringIO
 
+import pygments.styles
+
+
+from traitlets.config import Configurable
+from traitlets import Unicode, Bool
+
+
+available_themes = lambda : [s for s in pygments.styles.get_all_styles()]+['NoColor','LightBG','Linux']
+
+class Colorable(Configurable):
```

Comment:
If there's no existing module that makes sense for it (this one doesn't as far as I can see), then yes. But also, I guess there will be some more machinery that can be added to it later.